noise_raster/raster_processing.py

```diff
@@ -66,20 +56,68 @@ def sum_sound_level_3D(sound_levels: np.array):
     if len(sound_levels.shape) != 3:
         raise ValueError('Input array not 3D ')
 
-    sound_levels_fl32 = sound_levels.astype(np.float32)
+    sound_levels.astype(np.float32)
 
-    l, m, n = sound_levels_fl32.shape
+    l, m, n = sound_levels.shape
 
     sound_pressures = np.zeros((l, m, n)).astype(np.float32)
     sum_pressures = np.zeros((l, m, n)).astype(np.float32)
     out = np.zeros((l, m, n)).astype(np.float32)
 
-    sound_pressures = np.power(10, 0.1 * sound_levels_fl32)
+    sound_pressures = np.power(10, 0.1 * sound_levels)
     sum_pressures = np.sum(sound_pressures, axis=0)
-    out = 10 * np.log10(sum_pressures)
-    rounded_out = out.round(decimals=1)
+    out = (10 * np.log10(sum_pressures)).round(decimals=1)
+
+    return out
+
+def create_temp_directory():
+    """
+    Create a sub-directory in the current temp folder to hold intermediate files.
+    """
+    # Get current date and time
+    DATETIME = datetime.now()
+    # Convert date, time to string in format: dd_mm_YY_H_M_S
+    DATETIME_str = DATETIME.strftime("%d_%m_%Y_%H_%M_%S")
```

Comment:
Remarks on style (not a blocker for merging):

`DATETIME` shouldn't be capitalized as capitalized names are used for constants (e.g. in `constants.py`).

Also, the right side of the variable assignemt of `DATETIME` is not particularily complex and it's used only in the statement immediately following the assignment. In those cases, you could just write
```
datetime_str = datetime.now().strftime("%d_%m_%Y_%H_%M_%S")
```

Reply:
I will make this update.